lib/publiccloud/azure.pm

```diff
@@ -270,4 +276,11 @@ sub start_instance
     $instance->public_ip($self->get_ip_from_instance($instance));
 }
 
+
+
+sub create_instances {
```

Comment:
Is this change related to this commit? Why do we need this?

Reply:
ah you are right. It was during development that I changed the terraform apply timeout for Azure to 30min. But I guess it's not needed.  I will remove it!